test/system/movie_test.rb

```diff
@@ -0,0 +1,60 @@
+require "application_system_test_case"
+
+class MoviesTest < ApplicationSystemTestCase
+  test "admin can create new movie" do
+    sign_in users(:emina)
+    
+    visit root_path
+    click_on "Add New Movie"
+
+    fill_in "Title", with: "Test1"
+    select movies(:superman).rating, from: 'movie_rating'
+    fill_in "Total gross", with: movies(:superman).total_gross
+    fill_in "Released on", with: movies(:superman).released_on
+    fill_in "Description", with: movies(:superman).description
+    fill_in "Image file name", with: movies(:superman).image_file_name
+    check 'Action'
+
+    click_on "Create Movie"
+
+    assert_text "Movie successfully created!"
+  end
+
+  test "admin can't create movie, short description" do
+    sign_in users(:emina)
+    
+    visit root_path
```

Comment:
You don't need to go through this route(`root -> click_on button`) for each test 1 test is enough, for the performance reasons on the tests `23 and 42` you can just use `visit new_move_path`